src/refreshData.js

```diff
@@ -0,0 +1,16 @@
+const loadData = async () => {
+  fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/HuDs7U0CQZGKymImVaSp/scores/')
+    .then((response) => response.json())
+    .then((data) => {
+      const container = document.querySelector('.recent__container');
+      let dataToLoad = '';
+      for (let i = 0; i < data.result.length; i += 1) {
+        dataToLoad += `
+          <div class="recent__score">Name:${data.result[i].user} ${data.result[i].score}</div>    
+          `;
+      }
+      container.innerHTML = dataToLoad;
+    });
+};
+
+export default loadData;
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, according to the requirements you should use `async` and `await` features instead of `.then` and `.catch` to consume the API. Please consider refactoring this code! 🤓 🔧 

- [ ] **[OPTIONAL]:** Also, I suggest you use the `forEach` loop instead of the regular `for` one because it can increase the readability of your code tremendously. 🤓 🔧 

```suggestion
const loadData = async () => {
  fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/HuDs7U0CQZGKymImVaSp/scores/')
    .then((response) => response.json())
    .then((data) => {
      const container = document.querySelector('.recent__container');
      let dataToLoad = '';
      data.forEach((item) => {
        dataToLoad += `
          <div class="recent__score">Name:${item.user} ${item.score}</div>    
          `;
      })
      container.innerHTML = dataToLoad;
    });
};

export default loadData;
```